### ISSUE TITLE
爆块只收到0.25，剩下1.75丢失了，什么原因，solo的没加矿池

### ISSUE THREAD

**Issue by wangyao11sui** [Flax-Network/flax-blockchain#38]
爆块只收到0.25，剩下1.75丢失了，什么原因，solo的没加矿池

**Comment by ldsxp** [comment]
Flax 0.1.1 或者以上版本支持 nft，由于池化协议的限制，一周后才会收到剩下的 1.75。您可以在 https : //flaxexplorer.org/nft 输入 NFT 启动器 ID，查看这些奖励何时会自动进入钱包。

**Comment by flax-dev** [comment]
Yes, Flax 0.1.1 supports full payment of 2 XFX for NFT plots. You will need to wait 7 days before it is paid out (this limitation will be removed in Flax 0.2 which will be a mandatory upgrade when released). Please see https://flaxexplorer.org/nft to track the pending rewards.

是的，Flax 0.1.1 支持为 NFT 地块全额支付 2 XFX。 您需要等待 7 天才能支付（此限制将在 Flax 0.2 中删除，这将是发布时的强制性升级）。 请参阅 https://flaxexplorer.org/nft 以跟踪待处理的奖励。

**Comment by wangyao11sui** [comment]
信息已收到，非常感谢百忙之中抽空回复我的邮件，祝工作顺利，flax越来越好！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Flax-Network/flax-blockchain"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月25日(星期三) 晚上7:10
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Flax-Network/flax-blockchain] 爆块只收到0.25，剩下1.75丢失了，什么原因，solo的没加矿池 (#38)





 
Yes, Flax 0.1.1 supports full payment of 2 XFX for NFT plots. You will need to wait 7 days before it is paid out (this limitation will be removed in Flax 0.2 which will be a mandatory upgrade when released). Please see https://flaxexplorer.org/nft to track the pending rewards.
 
是的，Flax 0.1.1 支持为 NFT 地块全额支付 2 XFX。 您需要等待 7 天才能支付（此限制将在 Flax 0.2 中删除，这将是发布时的强制性升级）。 请参阅 https://flaxexplorer.org/nft 以跟踪待处理的奖励。
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
